config.xml

```diff
@@ -7,6 +7,7 @@
     <description>Jellyfin for Samsung Smart TV (Tizen).</description>
     <feature name="http://tizen.org/feature/screen.size.all"/>
     <icon src="icon.png"/>
+    <tizen:metadata key="http://tizen.org/metadata/app_ui_type/base_screen_resolution" value="extensive"/>
```

Comment:
According to https://github.com/jellyfin/jellyfin-tizen/pull/315#issuecomment-2544070043
```suggestion
    <tizen:metadata key="http://tizen.org/metadata/app_ui_type/base_screen_resolution" value="fullscreen"/>
```